scripts/supercmo_skills/stitch.py

```
@@ -67,6 +67,40 @@ def _probe(path):
 
+def _dur(path):
+    """Duration in seconds, or None."""
+    return _probe(path)[0]
+
+
+def _build_narration(ffmpeg, takes, clip_durations, workdir):
+    """One continuous narration track aligned to the clips: take N starts where clip N starts, and is
+    padded with silence to that clip's length. Returns (path, None) | (None, error). A take longer than
+    its clip is an error, not a truncation — the fix is a shorter line, never a faster read."""
+    padded = []
+    for i, (take, clip_len) in enumerate(zip(takes, clip_durations)):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)